### ISSUE TITLE
Unable to update any module

### ISSUE THREAD

**Issue by SoulCaliburSK** [PrestaShop/PrestaShop#33125]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hello,

I am unable to update any module in my PS. In the console I can see the following error:

Access to XMLHttpRequest at 'https://mbo-api.prestashop.com/api/shops/action' from origin 'https://mbo.prestashop.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

and

POST https://mbo-api.prestashop.com/api/shops/action net::ERR_FAILED 502

Using prestashop 8.1.0 now but the situation was same on 8.0.3 and 8.0.4 as well.

Not sure if this is any helpful but when I open https://mbo.prestashop.com I see:

![image](https://github.com/PrestaShop/PrestaShop/assets/138789282/2e719c35-be17-406f-8e13-b812b4e19873)

When I click on Modules list or Recommendation all I see is:

![image](https://github.com/PrestaShop/PrestaShop/assets/138789282/a47df8a4-1b8a-4694-b422-4ddd50771f6c)

Also I had this error when trying to open marketplace: AdminPsMboModule missing or invalid

I fixed the above error by copying the module to the modules as "ps_mbo" downloaded from https://addons.prestashop.com/en/administrative-tools/39574-prestashop-marketplace-in-your-back-office.html
version 4.3.3. The Marketplace now works but I am unable to update any of the modules. When I updated from 8.0.4 to 8.1.0 recently again I saw only AdminPsMboModule missing or invalid, again fixed by reinstalling the module by copying to FTP, then installing via module manager in BO. Tried to install also via upload module - result same.

Thanks to everyone who will try to help in advance.

### Expected behavior

Expect to be able to update modules.

### Steps to reproduce

Go to Modules - module manager - try to update any module - result:Could not perform action update for module undefined

In console there are errors:

Access to XMLHttpRequest at 'https://mbo-api.prestashop.com/api/shops/action' from origin 'https://mbo.prestashop.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

and

POST https://mbo-api.prestashop.com/api/shops/action net::ERR_FAILED 502

### PrestaShop version(s) where the bug happened

8.1.0

### PHP version(s) where the bug happened

Tried 7.4 and 8.0

### If your bug is related to a module, specify its name and its version

ps_mbo

**Comment by aniszr** [comment]
Hello,

Thanks for the issue, unfortunately I couldn't reproduce it.

Upgrade Module works well in both PS versions 8.0.4 & 8.1.0.

See attached videorecord:

https://github.com/PrestaShop/PrestaShop/assets/92626242/b67a7a51-9c2f-4aed-a55f-502beb71c914

Thanks!

**Comment by SoulCaliburSK** [comment]
Thank you for the answer, please let me know if there is anything I can try or anything I can provide in order to solve this.

My PS is still getting denied access to the module updates by CORS policy.

Thank you.

**Comment by kpodemski** [comment]
Hello @SoulCaliburSK 

The marketplace in back office module is not part of the PrestaShop Project. Maybe you could create a topic on the official PrestaShop forum or contact MBO teams through PrestaShop Marketplace.

**Comment by Jarode01** [comment]
Hi,

I'm facing **exactly** the same issue.
@SoulCaliburSK, Did you succeed to solve this ?

Thanks
